packages/dx-vue-grid/docs/guides/searching.md

```diff
@@ -0,0 +1,31 @@
+# Vue Grid - Searching
+
+The Grid component supports searching data programmatically or using the value an end user types in the corresponding Search Panel editor.
+
+## Related Plugins
+
+The following plugins implement the searching feature:
+
+- [SearchState](../reference/search-state.md) - controls the search state
+- [IntegratedFiltering](../reference/integrated-filtering.md) - performs built-in data searching and filtering
+- [SearchPanel](../reference/search-panel.md) - renders the search panel
```

Comment:
Should we mention toolbar here? We do it in the [Column Visibility](https://devexpress.github.io/devextreme-reactive/react/grid/docs/guides/column-visibility/) guide.

Reply:
Yes, we should